src/coreclr/jit/gentree.cpp

```diff
@@ -27054,6 +27057,13 @@ ClassLayout* GenTreeHWIntrinsic::GetLayout(Compiler* compiler) const
         case NI_AdvSimd_Arm64_LoadAndReplicateToVector128x4:
             return compiler->typGetBlkLayout(64);
 
+        case NI_Sve_LoadVectorx2:
+            return compiler->typGetBlkLayout(compiler->getVectorTByteLength() * 2);
```

Comment:
@kunalspathak - I think this is right? Unless we shouldn't be calling `GetLayout` for intrinsics that use `Vector<T>`.